src/pages/Reimbursement.js

```diff
@@ -1,150 +1,244 @@
 import React from "react";
-import { Button, Container, Form, FormGroup, FormText, Input, Label, FormFeedback } from "reactstrap";
+import {
+  Button,
+  Container,
+  Form,
+  FormGroup,
+  FormText,
+  Input,
+  Label,
+  FormFeedback
+} from "reactstrap";
 import "./Reimbursement.css";
 
-export default class ReimbursementForm extends React.Component{
-	constructor(props) {
-		super(props);
-		this.state = {
-			event: '',
-			description: '',
-			payTo: '',
-			amount: '',
-			file: '',
-			mail: true,
-			sofs: 'off',
-			eventValid: true,
-			descriptionValid: true,
-			payToValid: true,
-			amountValid: true,
-			fileValid: true,
-			deliveryFormatValid: true,
-		}
-		this.handleChange = this.handleChange.bind(this);
-		this.handleSubmit = this.handleSubmit.bind(this);
-	}
+export default class ReimbursementForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      event: "",
+      description: "",
+      payTo: "",
+      amount: "",
+      file: "",
+      mail: true,
```

Comment:
Sorry if I'm missing something here -- you have mail as a boolean here, but appear to be treating it like a string "on"/"off" later on? I'm a little confused on what's happening.

For the record I would prefer the boolean approach

Reply:
For some reason the radio buttons are weird. Starting it with on/off doesn't work, but updating it later with booleans doesn't work. This lets one of them start checked.